data/api/client-server/keys.yaml

```diff
@@ -288,7 +288,17 @@ paths:
   /keys/claim:
     post:
       summary: Claim one-time encryption keys.
-      description: Claims one-time keys for use in pre-key messages.
+      description: |-
+        Claims one-time keys for use in pre-key messages.
+
+        The request contains the user ID, device ID and algorithm name of the
+        keys that are required. The response contains a key matching this -
+        either a one-time key, or if none are available, a fallback key.
```

Comment:
How about:

>         The request contains the user ID, device ID and algorithm name of the
>        keys that are required. If a key matching these requirements can be
>        found, the response contains it. The returned key is a one-time key
>        if one is available, and otherwise a fallback key.

?

Reply:
Should we also mention that the server must not send the same one-time key in response to any other request?

Reply:
> How about:
> 
> > ```
> >     The request contains the user ID, device ID and algorithm name of the
> >    keys that are required. If a key matching these requirements can be
> >    found, the response contains it. The returned key is a one-time key
> >    if one is available, and otherwise a fallback key.
> > ```
> 
> ?

SGTM

Reply:
> Should we also mention that the server must not send the same one-time key in response to any other request?

As in, each OTK must be returned at most once? Yes, we probably should say that.

Reply:
> SGTM

10df9071

Reply:
> As in, each OTK must be returned at most once? Yes, we probably should say that.

b43fa770